### DISCUSSION TITLE
Remix Vite on Vercel

### DISCUSSION THREAD

**Discussion by 684efs3** [vercel/community#4729] [Bug Report]
### Summary

Right now deploying the latest version of Remix (built on Vite) does not work on Vercel. I get this error:

SyntaxError: Named export 'unstable_vitePlugin' not found. The requested module 'file:///vercel/path0/site/node_modules/@remix-run/dev/dist/index.js' is a CommonJS module, which may not support all module.exports as named exports.

It appears that Vercel has its own branch of Remix that does not reflect the Remix version in my package.json. Could you please fix this? 

Second, is it possible to deploy Remix as a Vite app now that Remix is a Vite plugin? Thank you!

### Steps to Reproduce

https://remix.run/blog/remix-heart-vite

npx create-remix@latest --template remix-run/remix/templates/unstable-vite

**Comment by amyegan** [comment]
Thanks for reporting this! It's already on our radar 🙂 https://github.com/vercel/remix/issues/63

**Reply by 684efs3** [reply]
Great!

**Reply by tombohub** [reply]
Is there any news on this? Thank you

**Reply by amyegan** [reply]
Please follow the open issue for the latest updates 🙏 

https://github.com/vercel/remix/issues/63
